nifi-toolkit/nifi-toolkit-admin/pom.xml

```diff
@@ -57,7 +57,7 @@ language governing permissions and limitations under the License. -->
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-web-security</artifactId>
-            <version>${client.version}</version>
+            <version>${project.version}</version>
```

Comment:
I would recommend avoiding these variables here and instead just use '1.2.0-SNAPSHOT'.  The former has proven problematic during the release process and the latter has proven highly reliable and gets automatically updated by the versions plugin during the release process.